app/src/main/java/com/cornellappdev/coffee_chats_android/SchedulingActivity.kt

```diff
@@ -155,6 +139,32 @@ class SchedulingActivity :
         setUpCurrentPage()
     }
 
+    override fun onNewIntent(intent: Intent?) {
+        super.onNewIntent(intent)
+        setUpDrawerLayout()
+    }
+
+    private fun setUpDrawerLayout() {
+        CoroutineScope(Dispatchers.Main).launch {
+            val getUserEndpoint = Endpoint.getUser()
+            val userTypeToken = object : TypeToken<ApiResponse<User>>() {}.type
+            val user = withContext(Dispatchers.IO) {
+                Request.makeRequest<ApiResponse<User>>(
+                    getUserEndpoint.okHttpRequest(),
+                    userTypeToken
+                )
+            }!!.data
+            drawerLayout.user_name.text =
+                getString(R.string.user_name, user.firstName, user.lastName)
+            drawerLayout.user_major_year.text = getString(
+                R.string.user_major_year,
+                user.major,
+                "'${user.graduationYear?.substring(2)}"
```

Comment:
Probably could put that `'` in the template string itself, rather than that bit of string manipulation.